spec/commands/generator/migration_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'pliny/commands/generator'
+require 'pliny/commands/generator/migration'
+require 'spec_helper'
+
+describe Pliny::Commands::Generator::Base do
+  subject { Pliny::Commands::Generator::Migration.new(migration_name, {}, StringIO.new) }
+
+  describe '#name' do
+    shared_examples 'a migration name' do
```

Comment:
We tend to avoid these clever but somewhat magical rspec features, at least when the simpler option seem just as readable/terse. Thoughts/Would you mind tweaking this test?